libafl/src/stages/calibrate.rs

```diff
@@ -365,6 +371,29 @@ where
             track_stability: false,
             restart_helper: ExecutionCountRestartHelper::default(),
             phantom: PhantomData,
+            name: Cow::Borrowed(CALIBRATION_STAGE_NAME),
         }
     }
+    /// Create a new [`CalibrationStage`], naming it as expected by AFL++.
+    #[must_use]
+    pub fn with_afl_name<F>(map_feedback: &F) -> Self
```

Comment:
this too.
we don't need another constructor. you can just use afl name everywhere